src/main/java/gov/cabinetofice/gapuserservice/web/LoginController.java

```diff
@@ -92,8 +95,25 @@ public RedirectView logout(
         return new RedirectView(colaLogout);
     }
 
+    @GetMapping("/refresh-token")
+    public ResponseEntity<String> refreshToken(@CookieValue(USER_SERVICE_COOKIE_NAME) final String currentToken, final HttpServletResponse response) {
+
+        jwtBlacklistService.addJwtToBlacklist(currentToken);
```

Comment:
Currently the token provided here is not validated, but as discussed once the security boilerplate PR is merged in it should check the token is valid before refreshing it